spec/fixtures/pr-diff.diff

```diff
@@ -0,0 +1,14 @@
+diff --git a/app/build.gradle b/app/build.gradle
```

Comment:
Using this PR https://github.com/dimagi/commcare-android/pull/2500/files for diff